src/abyssal_tome/utils.py

```
@@ -5,13 +5,14 @@
 def debounce(wait):
     """
     Decorator that debounces an asynchronous function, ensuring it is called only once after a specified delay.
-    
+
     Parameters:
         wait (float): The delay in seconds to wait after the last call before executing the function.
```

Ruff (0.11.9)

Line too long (101 > 100)

(E501)